tests/serverpod_test_server/test/serialization_test.dart

```diff
@@ -221,4 +223,55 @@ void main() {
 
     expect(typesMap.anObjectKey, {});
   });
+
+  group(
+    "Given an unknown type for protocol, ",
+    () {
+      test(
+          'when trying to deserialize using server protocol, then a SerializationTypeNotFoundException exception is thrown',
```

Comment:
Suggestion: Create a class in this test file with a name that makes it clear that it is unknown.

That way, if we start supporting `Object` for some reason this will still continue to validate what we would like to validate.